pyproject.toml

```diff
@@ -37,6 +37,11 @@ dependencies = [
     "partd >= 1.2.0",
     "pyyaml >= 5.3.1",
     "toolz >= 0.10.0",
+    # importlib.metadata has the following bugs fixed in 3.10.9 and 3.11.1
+    # https://github.com/python/cpython/issues/99130
+    # https://github.com/python/cpython/issues/98706
+    # TODO: when Python 3.12 is support is added this could be a
+    # conditional dependency
```

Comment:
If it works on all python versions >=3.11.1, why wait until the 3.12 bump to add a conditional dependency?